modules/installing-ocp-agent-manifest-folder.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-ocp-agent-manifest-folder_{context}"]
+= Creating a directory to contain additional manifests
+
+If you create additional manifests to configure your Agent-based installation beyond the `install-config.yaml` and `agent-config.yaml` files, you must create an `openshift` subdirectory within your installation directory.
+All of your additional machine configurations must be located within this subdirectory.
```

Comment:
Not sure what the style guide says about this, but I think if we find a way to explicitly mention that the most common type of extra manifest to add will be `MachineConfig` objects, then that will be more searchable.

I'm in two minds about whether we should also link to https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html#using-machineconfigs-to-change-machines since that describes a bunch of things you _can_ add MachineConfigs for on Day 1, but talks about applying them to the cluster with `oc apply` on Day 2.

Reply:
@zaneb I ended up going through with adding a link to the section you mentioned in your comment. The best case scenario would be having docs explicitly about adding `MachineConfig` objects on Day 1. 

But in absence of that sort of doc, I think it's better to link to these Day 2 docs and let users "synthesize" instructions about how to add the manifests during an Agent install, rather than making them figure out the entirety of the process on their own.